src/customizations/volto-form-block/components/View.jsx

```diff
@@ -209,7 +212,11 @@ const View = ({ data, id, path }) => {
             field: name,
             message: intl.formatMessage(messages.invalidEmailMessage),
           });
-        } else if (isBCC && !formData[name].otp) {
+        } else if (
+          data.email_otp_verification &&
+          isBCC &&
+          !formData[name].otp
```

Comment:
Questo check e' safe o `formData[name]` ha la possibilita' remota di esser undefined in qualche momento del suo ciclo di vita, e rompersi poi quando tenti di accedere a otp?

Reply:
è safe perchè viene gia fatto un controllo alla riga 208